controllers/idle_handlers.go

```diff
@@ -18,15 +18,19 @@ package controllers
 
 import (
 	"context"
+	"fmt"
 
 	ranv1alpha1 "github.com/openshift-kni/lifecycle-agent/api/v1alpha1"
+	"github.com/openshift-kni/lifecycle-agent/controllers/utils"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
 func (r *ImageBasedUpgradeReconciler) handleAbort(ctx context.Context, ibu *ranv1alpha1.ImageBasedUpgrade) (ctrl.Result, error) {
-
-	// TODO actual steps
-	// If succeeds, return doNotRequeue
+	ostreeUndeployCommand := fmt.Sprintf("ostree admin undeploy rhcos_%s", ibu.Spec.SeedImageRef.Version)
+	utils.ExecuteChrootCmd(utils.Host, ostreeUndeployCommand)
+	removeSysrootFilesCommand := fmt.Sprintf("unshare -m /bin/sh -c \"mount -o remount,rw /sysroot && rm -rf /sysroot/ostree/deploy/rhcos_%s\"", ibu.Spec.SeedImageRef.Version)
+	utils.ExecuteChrootCmd(utils.Host, removeSysrootFilesCommand)
+	utils.ExecuteChrootCmd(utils.Host, "rm -rf /var/ibu/")
```

Comment:
There is a constant in controllers/utils/cmd.go for this path /var/ibu, named Path (which should really be renamed)

Reply:
updated. changed to IBUWorkspacePath.